### PR TITLE
Update docs for service monitoring support

### PR DESCRIPTION
**Pull request by alam-zenoss**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -274,12 +274,43 @@ as devices with the following command.
...
++---------------------------+------------------------------------------+
+| Name                      | Description                              |
++===========================+==========================================+
+| zLinuxServicesModeled     | Accepts regular expressions that         |
+|                           | matches one or more services to model    |
++---------------------------+------------------------------------------+
+| zLinuxServicesNotModeled  | Accepts regular expressions that         |
+|                           | matches one or more services to not model|
++---------------------------+------------------------------------------+
+
```
Comment (pcarinhas) [inline README.rst]: Its probably better to put all the zProperties in a singe table in the configuraiton section, and then mention each of those properties in the sections that refer to them. This way the user has a single place to look at those.
You should also put the default values (if any) in that table as well so users know what is what.

A lot of our docs grow organically and it gets a bit choppy to read them, so this is good reason to consolidate some of this info.

Comment (pcarinhas) [reply README.rst]: Also, not part of your ticket: zIgnoreUnmounted is defined in LM but not mentioned in the docs.

File: README.rst [hunk 2]
```diff
@@ -274,12 +274,43 @@ as devices with the following command.
 
    zenbatchload <filename>
 
+Modeling and Monitoring OS Services
+-----------------------------------
+The Linux OS services are modeled using the *zenoss.cmd.linux.os_service* modeler plugin. The following systems are supported:
```
Comment (pcarinhas) [inline README.rst]: Please wrap these lines as per flow of the other sections

File: README.rst [hunk 3]
```diff
@@ -274,12 +274,43 @@ as devices with the following command.
...
++===========================+==========================================+
+| zLinuxServicesModeled     | Accepts regular expressions that         |
+|                           | matches one or more services to model    |
++---------------------------+------------------------------------------+
+| zLinuxServicesNotModeled  | Accepts regular expressions that         |
+|                           | matches one or more services to not model|
++---------------------------+------------------------------------------+
+
+Only *loaded* services are modeled. By default, both zProperties are ``<empty>``. An ``<empty>`` value in ``zLinuxServiceModeled`` is treated as ``.*`` regex and models all loaded services. Both the zProperties can support multiple regex expressions when separated on new lines. The *OSService* monitoring template generates events on every collection cycle for a service that is down. The events are automatically cleared if the service is up again.
+
```
Comment (pcarinhas) [inline README.rst]: Replace "By default,....bla bla,  when separateed on new lines" with

zLinuxServicesModeled and zLinuxServicesNotModeled are lists that accept regular expressions, and are by default empty.

File: README.rst [hunk 4]
```diff
@@ -274,12 +274,43 @@ as devices with the following command.
...
 
+Modeling and Monitoring OS Services
+-----------------------------------
+The Linux OS services are modeled using the *zenoss.cmd.linux.os_service* modeler plugin. The following systems are supported:
+
+- systemd (RHEL 7)
+- upstart (RHEL 6)
+- systemV (RHEL 5 and earlier)
+
+Version 2.3.0 and beyond provides support for monitoring the status of the services every collection cycle. However, only **systemd** and **systemV** are supported for monitoring. The following two zProperties filter the services that are to be modeled and thereby monitored.
```
Comment (pcarinhas) [inline README.rst]: Replace the entire paragraph with :

with

`Version 2.3.0  supports monitoring of the status of **systemd** and **systemV** system services. The zProperties 
*zLinuxServicesModeled* and *zLinuxServicesNotModeled* restrict the services that are modeled and monitored:`

File: README.rst (hunk 1 above)
Comment (alam-zenoss) [reply README.rst]: As we discussed in our chat, this will remain the same.

**Pull request merged**